source/modules/MEI.visual.xml

```diff
@@ -788,6 +787,12 @@
           <rng:ref name="data.BOOLEAN"/>
         </datatype>
       </attDef>
+      <attDef ident="sig.show" usage="opt">
```

Comment:
I made a couple comments before here, but I'm not sure where they went... so apologies if you are getting spammed!

Looking at this, I understand that `sig.show` is supposed to correspond with `keysig.show`. But how does `@sig.show` align with `@visible` which is already available on `<keySig>`?

Reply:
Ah, after seeing the issue mentioned in the commit message -- I'm not sure I would agree with `@sig.show`... we tend to use the word "visible", `@visible` in MEI, and AFAIK this is the only place where we're using "show". 

So perhaps, instead, it should be `keysig.visible` instead of `keysig.show`?